src/main/java/org/folio/inventory/support/JsonHelper.java

```diff
@@ -28,9 +28,11 @@ public static String getString(JsonObject representation, String propertyName) {
     return null;
   }
 
-  public static void includeIfPresent(JsonObject target, String propertyName, String value) {
-    if (target != null && StringUtils.isNotBlank(propertyName) && value != null) {
-      target.put(propertyName, value);
+  public static void includeIfPresent(
+    JsonObject representation, String propertyName, String value) {
```

Comment:
Why has the formatting changed to include all of the parameters on the following line?

Reply:
yes

Reply:
Apologies, I was asking why it has been formatted that way, rather than splitting the line part way through the parameters?

Reply:
It is because of the horizontal alignment.